inc/Calculator.h

```diff
@@ -11,6 +11,11 @@
 #define CALCULATOR_H
 
 #include <iostream>
-// Add your function declarations here.
+
+//Method to call from main. Loops until error/state > 0
+void RunCalculator(); 
```

Comment:
Every function declared in the header must have a doxygen comment. An example with params and return.
    /**
     * @brief Loads the configuration for this component instance
     *
     * @param configurationService the pointer to the system's configuration service
     * @return the configuration values to be used by this instance, according to the system's product and proto
     *         versions or nullptr if the configuration is not available for this instance.
     */